### ISSUE TITLE
Not good results for AgeCGANs

### ISSUE THREAD

**Issue by shivendrapsingh** [PacktPublishing/Generative-Adversarial-Networks-Projects#10]
Hi,
I trained all the models for AgeCGAN with 5000 images only, 500 epochs. I got reconstruction loss upto 0.20. But when i predict on new images, i don't get good results (not even close). Also i get very much the same predicted image for different test images.
Some help will be appreciated.
Thanks

**Comment by kailashahirwar** [comment]
Can you share your results here?

**Comment by shivendrapsingh** [comment]
![generated_image11](https://user-images.githubusercontent.com/8150963/66261159-cf5d8980-e7c8-11e9-96de-c1695489d88b.png)

This is the result i get (same result only few pixels different) for different facial images. I first use encoder.predict() on the resized image and then feed the encoder vector to generater.predict() function.

**Comment by curryJ** [comment]
I have the same question too.
I find that when I use the 'loaded_image = image.load_img(os.path.join(data_dir, image_path), target_size=image_shape)' ,I get the blur image.I think the  question is the image_shape(64,64,3) 
![image](https://user-images.githubusercontent.com/23001027/66709505-5ea20880-ed98-11e9-8a5e-bedd9ee96b86.png)
this is the original image
![image](https://user-images.githubusercontent.com/23001027/66709507-71b4d880-ed98-11e9-8f1f-97bb3f96ad78.png)

**Comment by kailashahirwar** [comment]
@shivendrapsingh As I see, your results are not good. Can you share your hyperparameters?

**Comment by kailashahirwar** [comment]
@curryJ I didn't get your problem?

**Comment by curryJ** [comment]
Could you get your result and hyperparameters?Thx

**Comment by curryJ** [comment]
@kailashahirwar Could you get your result and hyperparameters?Thx

**Comment by curryJ** [comment]
![image](https://user-images.githubusercontent.com/23001027/67622417-88553800-f84c-11e9-992b-60d777eee49c.png)
Could you tell me why the z_noise is selected  randomly? I konw it is for learning the distribution of the face-image data,but I'm thinking the z_noise is different in different epochs while the corresponding input_image is the same one.

**Comment by curryJ** [comment]
Hi,Thx your code for age-cGAN.But I train on wiki data which contains 60000 images,I get bad result.
This is my logs.
![image](https://user-images.githubusercontent.com/23001027/96809564-b8c6b680-144d-11eb-9180-5bffb17fbdde.png)
![image](https://user-images.githubusercontent.com/23001027/96809583-c3814b80-144d-11eb-9b32-eba0eecf5faa.png)

**Comment by deepesh-packt** [comment]
Please reopen the issue if the issue still persists.

**Comment by leminhds** [comment]
Sorry but what was the solution for this problem?? Lots of the issues here are closed without ever having an aswer
